packages/eslint-plugin/tests/rules/no-invalid-this.test.ts

```diff
@@ -0,0 +1,1188 @@
+import rule from '../../src/rules/no-invalid-this';
+import { RuleTester, getFixturesRootDir } from '../RuleTester';
+
+const rootPath = getFixturesRootDir();
+
+const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    sourceType: 'module',
+    tsconfigRootDir: rootPath,
+    project: './tsconfig.json',
```

Comment:
you're not using type information

```suggestion
const ruleTester = new RuleTester({
  parser: '@typescript-eslint/parser',
  parserOptions: {
    sourceType: 'module',
```

Reply:
I tried this, still the same !